cmd/basecoin/commands/init.go

```diff
@@ -64,7 +67,24 @@ func initCmd(cmd *cobra.Command, args []string) error {
 		return errors.New("Address must be 20-bytes in hex")
 	}
 
-	genesis := GetGenesisJSON(viper.GetString(FlagChainID), userAddr)
+	var options []string
```

Comment:
options and sep can be moved into the if statement as we only need them if options flags are provided.

Reply:
👍  addressed